pymemcache/test/test_client_retry.py

```diff
@@ -0,0 +1,315 @@
+
+import functools
```

Comment:
From a pep8 point of view I think that you could group the imports by lib types (stdlib first, then third libraries, and then the current lib)

Reply:
You're right. Didn't clean up properly after myself when I was writing tests.
https://github.com/martinnj/pymemcache/commit/03d7c48fba36934659fc37f1a4844da46d00f681